provider/openstack/cinder.go

```diff
@@ -56,7 +56,12 @@ func (env *Environ) cinderProvider() (*cinderProvider, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	return &cinderProvider{storageAdapter, env.name, env.uuid}, nil
+	return &cinderProvider{
+		storageAdapter,
```

Comment:
Why omit here **struct name fields**? You used below the name fields why here omit them? Why not always use them when you are returning a struct or a ptr to struct? I think this way, in my opinion,  are clearly specifying what values are attribute to, it's much more pleasing to the eye and more **consistent** (which is a good thing to have all around the juju codebase).